washee_app/lib/core/widgets/machine_card.dart

```diff
@@ -52,34 +52,33 @@ class MachineCard extends StatelessWidget {
               ),
               Spacer(),
               !machine.isAvailable
-                  ? 
-                    machine.activated
```

Comment:
Denne her er der merge conflict, men det er ren formattering, koden er 100% identisk, så du kan bare vælge det du synes ser pænest ud. (Dobbelt tjek dog at det ser ens ud, bare for en sikkerheds skyld!)